### PR TITLE
Add support for SDP v6.3 (API v20)

### PR DESCRIPTION
**Pull request by mandopaloooza**
## Description
- Add v20 CRDs
- Bump default API version to v20
- Update documentation

## Checklist
- [x] Bump `.version` and `.appVersion` in [k8s/crd/Chart.yaml](../k8s/crd/Chart.yaml)
- [x] Bump `.version` and `.appVersion` in [k8s/operator/Chart.yaml](../k8s/operator/Chart.yaml)

### CODE REVIEW

File: k8s/crd/templates/v20.yaml [hunk 1]
```diff
@@ -0,0 +1,2548 @@
+{{ if eq .Values.version "v20" }}
```
Comment (thomascellerier) [inline k8s/crd/templates/v20.yaml:1]: How was this file generated?
Might be worth documenting somewhere the process to add support for a new API version.

**Review by thomascellerier — APPROVED** [review]
Thanks!

File: k8s/crd/templates/v20.yaml (hunk 1 above)
Comment (thomascellerier) [reply k8s/crd/templates/v20.yaml:1]: Thanks!

**Comment by mandopaloooza** [comment]
- Add code to resolve supported entities given an API version (e.g. ClientConnection is deprecated in v20 but still available in v19 and v18) 
- Add load entity unit test for v19 and v20

**Pull request merged**